### ISSUE TITLE
Set link with arg & vars once restored

### ISSUE THREAD

**Issue by fontinixxl** [fontinixxl/moodle-qtype_programmedresp#25]
Once a restore of either a course or a quiz is done. We have to set for each question the linkervar it did before.

We need to get the quizid to store qtype_programmedresp_v_arg on backup.

**Comment by fontinixxl** [comment]
I'll need to post on moodle forum for help. I don't know how to deal with `backup::VAR_ACTIVITYID` from qtype backup.  It throw an error.

**Comment by fontinixxl** [comment]
Its is mark as wontfix because I don't find out the way to do that.

**Comment by fontinixxl** [comment]
Closed because the code will be refactored and this isn't valid anymore.
